tests/emulators/default_q0/platform.py

```diff
@@ -0,0 +1,48 @@
+import pathlib
+
+from qibolab.channels import ChannelMap
+from qibolab.instruments.emulator.pulse_simulator import PulseSimulator
+from qibolab.platform import Platform
+from qibolab.serialize import (
+    load_instrument_settings,
+    load_qubits,
+    load_runcard,
+    load_settings,
+)
+
+FOLDER = pathlib.Path(__file__).parent
+
+
+def create():
+    """Create a one qubit emulator platform."""
+
+    # load runcard and model params
+    runcard = load_runcard(FOLDER)
+
+    # Specify emulator controller
+    pulse_simulator = PulseSimulator()
+    instruments = {"pulse_simulator": pulse_simulator}
+    instruments = load_instrument_settings(runcard, instruments)
+
+    emulator_name = pulse_simulator.emulator_name
+
+    # extract quantities from runcard for platform declaration
+    qubits, couplers, pairs = load_qubits(runcard)
+    settings = load_settings(runcard)
+
+    # Create channel object
+    channels = ChannelMap()
+    channels |= (f"readout-{q}" for q in qubits.keys())
+    channels |= (f"drive-{q}" for q in qubits.keys())
+
+    # map channels to qubits
+    for q, qubit in qubits.items():
+        qubit.readout = channels[f"readout-{q}"]
+        qubit.drive = channels[f"drive-{q}"]
+
+        channels[f"drive-{q}"].qubit = qubit
+        qubit.sweetspot = 0  # not used
+
+    return Platform(
+        emulator_name, qubits, pairs, instruments, settings, resonator_type="2D"
```

Comment:
Fixed in 73777b3960a643a48cac3d0551c5239ffcb666e0. Now folder name, runcard['device_name'] and platform name are all the same.

Reply:
> or we should change the folder name to match the platform name, because otherwise qibocal complains.

@stavros11 could you kindly rerun the qibocal routines again and see if it still complains? It should be resolved now, if I understood your suggestion correctly.